ligand/templates/endogenous_browser.html

```diff
@@ -225,30 +227,32 @@
                         <!-- Receptor block -->
                         <td style="text-align:right">{{value.0}}</td>
                         <td style="text-align:right">{{value.1}}</td>
-                        <td style="text-align:right"><a target='_blank' href="http://www.uniprot.org/uniprot/{{value.18}}">{{value.2}}</a></td> <!-- UniProt -->
-                        <td style="text-align:right"><a target='_blank' href="/protein/{{value.19}}">{{value.3}}</a></td> <!-- IUPHAR -->
+                        <td style="text-align:right"><a target='_blank' href="http://www.uniprot.org/uniprot/{{value.20}}">{{value.2}}</a></td> <!-- UniProt -->
+                        <td style="text-align:right"><a target='_blank' href="/protein/{{value.21}}">{{value.3}}</a></td> <!-- IUPHAR -->
```

Comment:
This filter is not working correctly. Try selecting A1. It either needs the safe tag back or the filter type needs to be changed.